test/e2e/common/pods.go

```diff
@@ -369,7 +369,7 @@ var _ = framework.KubeDescribe("Pods", func() {
 		Testname: Pods, ActiveDeadlineSeconds
 		Description: Create a Pod with a unique label. Query for the Pod with the label as selector MUST be successful. The Pod is updated with ActiveDeadlineSeconds set on the Pod spec. Pod MUST terminate of the specified time elapses.
 	*/
-	framework.ConformanceIt("should allow activeDeadlineSeconds to be updated [NodeConformance]", func() {
+	framework.ConformanceIt("should allow activeDeadlineSeconds to be updated and pod must terminate with the status DeadlineExceeded [NodeConformance]", func() {
```

Comment:
This does not adhere to conformance [test-requirements](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/conformance-tests.md#conformance-test-requirements). 
>Conformance tests are intended to be stable and backwards compatible according to the standard API deprecation policies. Therefore any test that relies on specific output that is not subject to the deprecation policy cannot be promoted to conformance.

It would be good to update e2e along with the name, incase it is verifying important functional behaviour and stands as candidate for Conformance.

More insights are: https://github.com/kubernetes/kubernetes/issues/75324#issuecomment-489880420